iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.hpp

```diff
@@ -37,6 +37,7 @@ enum class AllocationError
     RUNNING_OUT_OF_CHUNKS,
     TOO_MANY_CHUNKS_ALLOCATED_IN_PARALLEL,
     INVALID_CHUNK,
+    INVALID_PARAMETER_FOR_CHUNK,
```

Comment:
PARAMETER or better INVALID_CHUNK_SETTINGS? What is the error used for?

Reply:
it's used for cases where invalid parameter are provided for the chunk e.g. payload alignment